evap/student/templates/student_vote.html

```diff
@@ -200,6 +221,18 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                 bottomCard: document.querySelector("#bottom_text_results_publish_confirmation_card"),
             };
 
+            // Ensure that selected questionnaire language is saved and loaded
+            const languageStorageKey = "student-vote-language-{{ evaluation.id }}-{{ request.user.id }}";
+            const params = new URLSearchParams(document.location.search);
+            const currentlySelectedLanguage = params.get("language") || "";
+            const savedLanguage = localStorage.getItem(languageStorageKey);
+            if (currentlySelectedLanguage) {
+                localStorage.setItem(languageStorageKey, currentlySelectedLanguage);
```

Comment:
These entries don't expire automatically, should we do something about it? @richardebeling

Reply:
Yes, also no blocker for me. Not sure how to best approach this, but either way, let's do something about it in a separate issue